custom-login/src/Login.jsx

```diff
@@ -46,20 +46,15 @@ const Login = () => {
     widget.renderEl(
```

Comment:
Should we be showing a recommended way other than renderEl?

Reply:
I saw Aaron has a separated PR for this https://github.com/okta/samples-js-react/pull/146
Let's merge from there.